snippets/findify-filter-selected.liquid

```diff
@@ -33,7 +33,7 @@
  
                 {% endcase %}
             {% else %}
-        <div class="findify-filters-breadcrumb-item" selected="true" type="{{type}}" name="{{name}}" value="{{value}}">
+        <div class="findify-filters-breadcrumb-item" selected="true" type="{{type}}" name="{% if name == 'color-text' %}color{% else %}{{name}}{% endif %}" value="{{value}}">
```

Comment:
Condition within the HTML can be done, but makes the HTML itself not readable. Either define the variable on top of this line of HTML or at the beginning of the file along the other stuff.